src/utils/string.js

```diff
@@ -13,17 +13,19 @@ export function removeURLs(
   string: string,
   removeExtensions: boolean = true
 ): string {
-  const regExpExtension = removeExtensions ? '|moz-extension' : '';
+  const rmExtensions = removeExtensions ? '|moz-extension:' : '';
   const regExp = new RegExp(
-    '((https?|ftp' + regExpExtension + ')://)([^\\s/$.?#].[^\\s)]*)',
-    // ^                                       ^           ^
-    // |                                       |           matches any characters except
-    // |                                       |           whitespaces and ) character.
-    // |                                       |           Other characters are allowed now
-    // |                                       matches any characters except whitespaces
-    // |                                       and / $ . ? # characters because this is
-    // |                                       start of the URL
-    // Matches http, https, ftp and optionally moz-extension protocols
+    '\\b((?:https?:|ftp:|file:/?' + rmExtensions + ')//)[^\\s/$.?#][^\\s)]*',
```

Comment:
I think that instead of doing `file:/?` here, you could do `file:` and put `/?` after the group `)`. It's not a big deal that would match things like `https:///`, as this doesn't occur anyway.

Reply:
I did like this because I didn't want to match the urls like `https:///`. I'm still not sure if it's worth doing but we have a unit test for that, that's why I didn't want to regress it:
https://github.com/firefox-devtools/profiler/blob/3c67af6709a85c5d65d9fa5700a6dfe2ce8ed733/src/test/unit/string.test.js#L114-L115